### DISCUSSION TITLE
Check Up for hamdivazim

### DISCUSSION THREAD

**Discussion by MKM12345** [MKM12345/Backend-Languages-editor#1] [General]
@hamdivazim, can you make sure that the files go like this? Thanks:
editor/
├── backend/
│   ├── editor.py
│   └── runner.py
├── frontend/
│   ├── index.html
│   ├── styles.css
│   └── scripts.js
└── server/
    ├── editor/
    │   ├── __init__.py
    │   ├── settings.py
    │   ├── urls.py
    │   └── views.py
    ├── manage.py
    ├── db.sqlite3
    ├── .env
    ├── requirements.txt

**Comment by MKM12345** [comment]
Make sure they're empty.

**Reply by MKM12345** [reply]
Also @hamdivazim, we may need to deploy this onto GitHub Pages so can you add a ```/docs``` directory? Thanks.

**Reply by hamdivazim-archive** [reply]
Ok will do. Also, is it fine if I just deploy from a different branch? That would make more sense to me

**Reply by MKM12345** [reply]
Ok, if you want.
